tests/manager/manager-set-uri-options-002.phpt

```diff
@@ -46,6 +46,6 @@ printf("Inserted: %d\n", $inserted);
 ===DONE===
 <?php exit(0); ?>
 --EXPECTF--
-OK: Got Exception
+OK: Got MongoDB\Driver\Exception\ConnectionTimeoutException
```

Comment:
At a glance, I'm not sure why failure to specify `ssl=true` when connecting to a server requiring SSL manifests itself as a ConnectionTimeoutException instead of a generic ConnectionException.

It may be helpful to add the exception message here (adding `echo` before `throws()`, followed by `, "\n";` afterwards). [PHPC-332](https://jira.mongodb.org/browse/PHPC-332) is quite old, so I don't recall the original context of the issue.